code/modules/mechanics/MechanicProcess.dm

```diff
@@ -0,0 +1,44 @@
+/// handles obj/item/mechanics/process()
+/datum/controller/process/mechanics
+	var/tmp/list/detailed_count
+	var/tmp/tick_counter
+	var/tmp/processing_mechanics
+
+	setup()
+		name = "Mechanics"
+
+		schedule_interval = 0.4 SECONDS
+
+		logTheThing(LOG_DEBUG, src, "Mechanics process intialized")
+
+		detailed_count = new
+
+		src.processing_mechanics = global.processing_mechanics
+
+	copyStateFrom(datum/controller/process/target)
+		var/datum/controller/process/mechanics/old_items = target
+		src.processing_mechanics = old_items.processing_mechanics
+		src.detailed_count = old_items.detailed_count
```

Comment:
Vars don't do anything useful and just waste memory duplicating a global list that we then ignore. copyStateFrom just perpetuates an unneeded and a blank list
```suggestion
```